### PR TITLE
/me endpoint

### PR DESCRIPTION
**Pull request by Podgore**

### CODE REVIEW

File: StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs [hunk 1]
```diff
@@ -9,4 +9,5 @@ public interface IUserService
 {
     Task<PageList<StudentDTO>> GetStudentsAsync(SearchRequest request);
     Task<UserDTO> UpdateUserAsync(Guid userId, UpdateUserDTO dto);
+    Task<UserDTO> GetUserAsync(Guid userId);
```
Comment (fedyafedko) [inline StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs:12]:
```suggestion
    Task<UserDTO> Me(Guid userId);
```

**Review by fedyafedko — CHANGES_REQUESTED** [review]

File: StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs (hunk 1 above)
Comment (fedyafedko) [reply StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs:12]: Why? You creat this service for profile. So i think it is better to call it "Me" for this service

Comment (Podgore) [reply StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs:12]: 1) where async?
2)we called method what it doing(especially in service). You calling method Me, what does he doing? Its clear to us what it is, for other developers - not

Comment (fedyafedko) [reply StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs:12]: U get information about user only for profile

Comment (aquaprogit) [reply StudyHub/StudyHub.BLL/Services/Interfaces/IUserService.cs:12]: I think service method named "me" doesn't clear define what method is actually doing. Keep it like GetUserById or smth like it was

**Review by fedyafedko — APPROVED** [review]

**Pull request merged**